cpp/src/binaryop/compiled/binary_ops.cu

```diff
@@ -16,11 +16,16 @@
 
 #include "binary_ops.hpp"
 #include "operation.cuh"
+#include "struct_binary_ops.cuh"
```

Comment:
The including headers should be added with full path.
For example, `struct_binary_ops.cuh` should be included as
```
#include <binaryop/compiled/struct_binary_ops.cuh>
```

Same apply for all other headers.